experimental/packages/opentelemetry-browser-detector/test/util.ts

```diff
@@ -17,6 +17,7 @@ import { Suite } from 'mocha';
 import * as assert from 'assert';
 import { BROWSER_ATTRIBUTES } from '../src/types';
 import { IResource } from '@opentelemetry/resources';
+import { DetectedResource } from '@opentelemetry/resources/src/types';
```

Comment:
Is this an intentional deep-import? 🤔

Reply:
My browser automatically added it. Looks like I forgot to export `DetectedResource` originally (and `DetectedResourceAttribute`).